buildSrc/src/main/java/io/opentelemetry/instrumentation/gradle/AutoInstrumentationPlugin.java

```diff
@@ -43,6 +51,40 @@ public void apply(Project project) {
             });
   }
 
+  private void addDependencies(Project project) {
```

Comment:
It seems the upstream does not have `AutoInstrumentationPlugin.java` in 1.3x anymore. 

Shall we align more with the upstream and remove this file as well?